setup.py

```diff
@@ -21,6 +28,11 @@ def c_files_in(directory):
             version = m.group(1)
 assert version
 
+class GPerfingBuildExt(build_ext):
+    def run(self):
+        os.system("gperf src/entities.gperf > src/entities.c")
```

Comment:
it might be easier just to spit out a `.h` and `#include` it like the pre-generated html_blocks stuff does.